docs/use_case/postv1.md

```diff
@@ -0,0 +1,77 @@
+This initial version of the demo #post app will allow you to create and visualise your own post(s). It will also showcase examples of the ONEM's python SDK usage.
+
+So let's navigate to the **postv1** folder and create/activate our virtual environment:
+
+```bash
+onem@local:~$ cd ~/postv1 ; pipenv shell --python 3.7
+```
+
+
+Also please make sure that:
+
+ - the database is created (even if we will store the posts in-memory).
```

Comment:
I suggest just to not mention the "database" word if it's not used.

Reply:
It gets automatically created when running the initial migrations(Django's internal stuff), but I'll rephrase this and remove the `database` word.

Reply:
Please note that the default is having an sqlite3 file and it is not in memory. Database is the right word to use here.

Reply:
@mvnm - in postv1 we store the newly created posts in-memory. In v2 and v3 we use sqlite.